src/styles/main.scss

```diff
@@ -0,0 +1,61 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+.stopwatch {
```

Comment:
check this checklist point
<img width="645" alt="image" src="https://user-images.githubusercontent.com/102411130/226276423-e02db801-bcf1-4f37-8c70-8c1133b6cc1a.png">